web_src/js/features/contributors.js

```diff
@@ -0,0 +1,28 @@
+import {createApp} from 'vue';
+
+export async function initRepoContributors() {
+  const el = document.getElementById('repo-contributors-chart');
+  if (!el) return;
+
+  const {default: RepoContributors} = await import('../components/RepoContributors.vue');
```

Comment:
```suggestion
  const {default: RepoContributors} = await import(/* webpackChunkName: "contributors-graph" */'../components/RepoContributors.vue');
```

Always good to have named chunks and it will let us verify easily in webpack's build output.